git/hooks/prepare-forward-merge

```diff
@@ -4,7 +4,7 @@ require 'net/http'
 require 'yaml'
 require 'logger'
 
-$main_branch = "2.5.x"
+$main_branch = "2.6.x"
```

Comment:
This was done in the past as well. I must admit, I'm not sure if this is correct.

Reply:
Thanks, @dreis2211. Your change is correct. Without it a forward merge into `main` would say `2.5.x` in its message rather than `2.6.x`. This would then cause the `forward-merge` hook to match against the wrong milestone on GitHub so the issue it creates would be in the wrong version.

Reply:
Ah, it's matching the milestone - that makes sense. Thanks for clarifying.